packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -466,25 +545,45 @@ class _DraggableScrollableSheetScrollPosition
     }
   }
 
+  bool get _isAtSnapSize =>
```

Comment:
Flutter style says not to use `=>` unless it's a short, one line function. There are some additional formatting issues below (it's ok to exceed 80 chars if it's more readable).